var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -140,6 +140,8 @@ def build_args(self, spec, prefix):
         return args
 
     def setup_environment(self, spack_env, run_env):
+        if not 'python' in self.spec:
```

Comment:
could you add #TODO here mentioning this is a temporary fix needed for external package